2021-10-13-github-templates.md

```diff
@@ -0,0 +1,170 @@
+    - [ ] Bug fix (non-breaking change which fixes an issue)
+    - [ ] New feature (non-breaking change which adds functionality)
+    - [ ] Breaking change (fix or feature that would cause existing functionality to change)
+
+    ## Checklist:
+    <!--- Go over all the following points, and put an `x` in all the boxes that apply. -->
+    <!--- If you're unsure about any of these, don't hesitate to ask. We're here to help! -->
+    - [ ] My code follows the code style of this project.
+    - [ ] My change requires a change to the documentation.
+    - [ ] I have updated the documentation accordingly.
+    - [ ] I have read the **CONTRIBUTING** document.
+    - [ ] I have added tests to cover my changes.
+    - [ ] All new and existing tests passed.
+    - [ ] I will abide by the code of conduct.
+```
+
+
+### Issues: Bug Reports
+
+```
+    <!--- Provide a general summary of the issue in the Title above -->
+
+    ## Before we start...
+
+    - [ ] I checked the documentation and found no answer
+    - [ ] I checked to make sure that this issue has not already been filed
+    - [ ] I'm reporting the issue to the correct repository (for multi-repository projects)
+
+    ## Description
+    <!--- Provide a more detailed introduction to the issue itself, and why you consider it to be a bug -->
+
+    ## Expected Behavior
+    <!--- Tell us what should happen -->
+
+    ## Actual Behavior
+    <!--- Tell us what happens instead -->
+
+    ## Possible Fix
+    <!--- Not obligatory, but suggest a fix or reason for the bug -->
+
+    ## Steps to Reproduce
+    <!--- Provide a link to a live example, or an unambiguous set of steps to -->
+    <!--- reproduce this bug. Include code to reproduce, if relevant -->
+    1.
+    2.
+    3.
+    4.
+
+    ## Context
+    <!--- How has this bug affected you? What were you trying to accomplish? -->
+
+    ## Your Environment
+    <!--- Include as many relevant details about the environment you experienced the bug in -->
+    * What branch/commit are you on?:
+    * Version used:
+    * Environment name and version (e.g. Chrome 39, node.js 5.4):
+    * Operating System and version (desktop or mobile):
+    * Link to your project:
+
+    **Screenshots and Videos**
+    <!--- If the issue has an effect in the frontend, include any relevant screenshots and videos here. -->
+
+    **Logs**
+    <!--- Include relevant log snippets or files here. -->
+
+    **I will abide by the [code of conduct] ()**
+```
+
+### Issues: Feature Requests
+
+```
+    <!--- Provide a general summary of the issue in the Title above -->
+
+    ## Before we start...
+    - [ ] I checked the documentation and didn't find this feature
+    - [ ] I checked to make sure that this feature has not already been requested
+
+    ## Detailed Description
+    <!--- Provide a detailed description of the change or addition you are proposing -->
+
+    ## Context
```

Comment:
This should be part of the detailed description of the feature, I don't think it needs its own section